api/src/controllers/server.ts

```diff
@@ -81,4 +88,113 @@ router.get(
 	respond
 );
 
+router.get(
+	'/schema/snapshot',
+	asyncHandler(async (req, res, next) => {
+		const service = new SchemaService({ accountability: req.accountability, schema: req.schema });
```

Comment:
We should not pass `schema` in so as to fetch a fresh schema that bypasses cache.

```suggestion
		const service = new SchemaService({ accountability: req.accountability });
```